### DISCUSSION TITLE
Zemismart jl-zb

### DISCUSSION THREAD

**Discussion by timkvakin** [Koenkk/zigbee2mqtt#11796] [Devices]
The link to the product:
https://www.zemismart.com/products/zemismart-jl-zb-downlight

The information when pairing the device:
MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"friendly_name":"0xa4c13891558bf343","supported":false},"type":"pairing"}'
Warning Received message from unsupported device with Zigbee model 'TS0505B' and manufacturer name '_TZ3210_leyz4rju'

Would it be possible to add it to the supported devices:

**Comment by Koenkk** [comment]
Already supported in the latest dev branch. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by timkvakin** [comment]
Really. Thx.

**Comment by Autogarsas** [comment]
Hi, may i ask if this is same product? As the link is dead from author.
https://www.zemismart.com/products/zemismart-jl-zb
